src/utils/output.ts

```diff
@@ -19,19 +19,23 @@ import {
   ComponentSet,
   CodeCoverage,
 } from '@salesforce/source-deploy-retrieve';
-import { NamedPackageDir } from '@salesforce/core';
-import { API, TestLevel } from './types';
+import { Messages, NamedPackageDir, SfProject } from '@salesforce/core';
+import { StandardColors } from '@salesforce/sf-plugins-core';
+import { API, AsyncDeployResultJson, DeployResultJson, RetrieveResultJson, TestLevel, Verbosity } from './types';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-deploy-retrieve', 'deploy.async');
 
 function info(message: string): string {
   return blue(bold(message));
```

Comment:
This deviates from StandardColors.info.

Reply:
@peternhale This is only being used for the titles on tables. I'll rename the function appropriately and keep it blue - unless you think there's a better color for that.